samples/subsys/usb/mass/boards/stm32l562e_dk_ns.overlay

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,code-partition = &slot1_partition;
+	};
+};
+
+&flash0 {
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00019000>;
+			read-only;
+		};
+		/* Secure image primary slot */
+		slot0_partition: partition@19000 {
+			label = "image-0";
+			reg = <0x00019000 0x0003C000>;
+		};
+		/* Non-secure image primary slot */
+		slot1_partition: partition@55000 {
+			label = "image-1";
+			reg = <0x00055000 0x0002A000>;
+		};
+	};
+};
+
+&sdmmc1 {
+	status = "okay";
+
+	pinctrl-0 = <&sdmmc1_d0_pc8
+		     &sdmmc1_d1_pc9
+		     &sdmmc1_d2_pc10
+		     &sdmmc1_d3_pc11
+		     &sdmmc1_ck_pc12
+		     &sdmmc1_cmd_pd2>;
+
+	pinctrl-names = "default";
+
+	cd-gpios = <&gpiof 2 GPIO_ACTIVE_LOW>;
+};
```

Comment:
Move this to board description (ns/s common part)

Reply:
Done!